test/AWS.Deploy.CLI.UnitTests/Utilities/TestToolOrchestratorInteractiveService.cs

```diff
@@ -8,12 +8,14 @@ namespace AWS.Deploy.CLI.UnitTests.Utilities
 {
     public class TestToolOrchestratorInteractiveService : IOrchestratorInteractiveService
     {
+        public IList<string> SectionStartMessages { get; } = new List<string>();
         public IList<string> DebugMessages { get; } = new List<string>();
         public IList<string> OutputMessages { get; } = new List<string>();
         public IList<string> ErrorMessages { get; } = new List<string>();
 
-        public void LogDebugLine(string message) => DebugMessages.Add(message);
-        public void LogErrorMessageLine(string message) => ErrorMessages.Add(message);
-        public void LogMessageLine(string message) => OutputMessages.Add(message);
+        public void LogSectionStart(string message, string? description) => SectionStartMessages.Add(message);
```

Comment:
Is the warning on GitHub here addressable - `The annotation for nullable reference types should only be used in code within a '#nullable' annotations context.`?

If so, think it'd be nice to keep build warnings clean, especially if this will appear on all future PRs.

Reply:
Removed. Didn't realize the test project didn't have nullablity turned on.